Source/Immutable/Public/Immutable/ImmutablePassport.h

```diff
@@ -87,11 +87,18 @@ class IMMUTABLE_API UImmutablePassport : public UObject
 	 * This will open the user's default browser and take them through Passport login. 
 	 * @param IsConnectImx 		If true, the "re-connect" method is used to authenticate into Passport with Immutable X.
 	 * 							Else, "re-login" is used for authentication. To access a wallet with Immutable X or zkEVM later, you must call "Connect" again with this value set to true, or use "ConnectEvm."
-	 * @param TryToRelogin 		If true, the game bridge will use a cached session to re-connect or re-login the user, avoiding the need for a web browser. If this attempt fails, it will fall back to device code authentication.
+	 * @param TryToRelogin 		If true, the game bridge will use a cached session to re-connect or re-login the user, avoiding the need to open a web browser. If this attempt fails, it will fall back to device code authentication.
 	 * @param ResponseDelegate 	Callback delegate.
 	 */
 	void Connect(bool IsConnectImx, bool TryToRelogin, const FImtblPassportResponseDelegate& ResponseDelegate);
 #if PLATFORM_ANDROID | PLATFORM_IOS | PLATFORM_MAC
+	/**
+	 * (Android, iOS and macOS only) Logs into Passport using Authorization Code Flow with Proof Key for Code Exchange (PKCE)
```

Comment:
```suggestion
	 * (Android, iOS and macOS only) Logs into Passport using Authorisation Code Flow with Proof Key for Code Exchange (PKCE)
```
Always write in British English